### PR TITLE
Fixed issue #389

### PR DESCRIPTION
**Pull request by thesusian**
Fixed issue #389 with dislikes count showing up as 0 on videos that have the like and dislike count hidden by the creator

### CODE REVIEW

**Comment by cupiditys** [comment]
sometimes says "dislikes disabled" even on videos with ratings on
image

**Comment by thesusian** [comment]
can you share the video link / browser, It's seems to be working fine for me.
tested on (Mozilla Firefox 95.0.2) and (Chromium 96.0.4664.110)

**Comment by cupiditys** [comment]
im not sure exactly what causes it since when i refreshed it was fixed but just happened randomly when i clicked on a video

**Comment by Anarios** [comment]
Remove /dist folder - it's in gitignore now.

**Review by PickleNik — CHANGES_REQUESTED** [review]
Remove the /dist folder from committed files please :)

**Comment by thesusian** [comment]
Removed /dist

File: Extensions/combined/src/bar.js [hunk 1]
```diff
@@ -1,48 +1,57 @@
...
+                    ></div>
+              </div>
+            </div>
+            <tp-yt-paper-tooltip position="top" id="ryd-dislike-tooltip" class="style-scope ytd-sentiment-bar-renderer" role="tooltip" tabindex="-1">
+              <!--css-build:shady-->${likes.toLocaleString()}&nbsp;/&nbsp;${dislikes.toLocaleString()}
+            </tp-yt-paper-tooltip>
+            </div>
+    `
+      );
+    } else {
```
Comment (Anarios) [inline Extensions/combined/src/bar.js:43]: Coule you please apply Prettier styles? Lots of changes with only spaces, hard to read a PR, thanks!

**Comment by PickleNik** [comment]
@Anarios

File: Extensions/combined/src/bar.js (hunk 1 above)
Comment (thesusian) [reply Extensions/combined/src/bar.js:42]: Done and commited

**Comment by Anarios** [comment]
Thanks!

**Pull request merged**